src/frameworks/web/components/atoms/SearchField/SearchField.tsx

```diff
@@ -0,0 +1,87 @@
+import styled from 'styled-components';
+import React from 'react';
+import theme from 'theme';
+import {
+  // eslint-disable-next-line no-unused-vars
+  _ISearchFieldContainer, ISearchField, ISearchFieldState,
+} from 'interfaces/frameworks/web/components/atoms/SearchField/ISearchField';
+
+const InputBox = styled.input`
+  width: 100%;
+  font-family: 'Muli', 'Noto Sans KR', sans-serif;
+  font-size: 16px;
+  font-weight: bold;
+  margin-left: 24px;
+  margin-right: 12px;
```

Comment:
storybook으로 확인해보니 vertical 정렬이 안맞던데 이런식으로 수동 정렬을 시키면 어려울까요?
```suggestion
  margin: 14px 12px 18px 24px;
```

Reply:
![Screen Shot 2020-03-25 at 12 43 41 AM](https://user-images.githubusercontent.com/10934461/77447606-33394480-6e33-11ea-9e89-478a95ffb662.png)

확인해보니, Safari 에서 잘리는 문제를 확인했어요!
`margin` 으로 하면 살짝 휴리스틱(?) 하기 때문에 `line-height` 로 조정했습니다~!